### PR TITLE
Add Multi-Level-Feedback-Queue scheduling algorithm

### PR DESCRIPTION
**Pull request by dongjji**
### Describe your change:

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: scheduling/multi_level_feedback_queue.py [hunk 1]
```diff
@@ -0,0 +1,319 @@
+from collections import deque
+
+
+class Process:
+    """
+    Process should have these fields below
+    """
```
Comment (poyea) [inline scheduling/multi_level_feedback_queue.py:7]: it's ok to remove

File: scheduling/multi_level_feedback_queue.py [hunk 2]
```diff
@@ -0,0 +1,319 @@
...
+        self.burst_time = burst_time  # remaining burst time
+        self.waiting_time = 0  # total time of the process wait in ready queue
+        self.turnaround_time = 0  # time from arrival time to completion time
+
+
+class MLFQ:
+    """
+    MLFQ(Multi Level Feedback Queue)
+    https://en.wikipedia.org/wiki/Multilevel_feedback_queue
+    This MLFQ has a lot of queue that has different priority
```
Comment (poyea) [inline scheduling/multi_level_feedback_queue.py:23]:
```suggestion
    MLFQ has a lot of queues that have different priority
```

File: scheduling/multi_level_feedback_queue.py [hunk 3]
```diff
@@ -0,0 +1,319 @@
...
+    In this MLFQ,
+    The first Queue(0) to last second Queue(N-2) of MLFQ has Round Robin Algorithm
```
Comment (poyea) [inline scheduling/multi_level_feedback_queue.py:25]:
```suggestion
    The first Queue(0) to last second Queue(N-2) of MLFQ have Round Robin Algorithm
```

File: scheduling/multi_level_feedback_queue.py [hunk 4]
```diff
@@ -0,0 +1,319 @@
...
+        >>> finish_queue, ready_queue = mlfq.round_robin(ready_queue, 25)
+        >>> mlfq.calculate_remaining_burst_time_of_processes(mlfq.finish_queue)
+        [0, 0]
+        >>> mlfq.calculate_remaining_burst_time_of_processes(ready_queue)
+        [11, 26]
+        """
+        remaining_burst_times = []
+        for i in range(len(queue)):
+            remaining_burst_times.append(queue[i].burst_time)
+        return remaining_burst_times
```
Comment (poyea) [inline scheduling/multi_level_feedback_queue.py:139]:
```suggestion
        return [q.burst_time for q in queue]
```

**Review by poyea — APPROVED** [review]
Thank you for your pull request!🤩 Looks good to me in general!!

File: scheduling/multi_level_feedback_queue.py (hunk 1 above)
Comment (dongjji) [reply scheduling/multi_level_feedback_queue.py]: Thanks for your suggestions!😃😃
I'm sorry for bothering you cause my English skill is not good.

**Comment by dongjji** [comment]
I'm sorry. `awaiting merge` label is removed because of my new commit.
I don't know if this is right.🥲

File: scheduling/multi_level_feedback_queue.py [hunk 5]
```diff
@@ -0,0 +1,313 @@
+from collections import deque
+
+
+class Process:
+
```
Comment (poyea) [inline scheduling/multi_level_feedback_queue.py:5]:
```suggestion
```

**Review by poyea — APPROVED** [review]
Thank you for your pull request!🤩

**Pull request merged**